pkg/api/defaults.go

```diff
@@ -8,9 +8,13 @@ import (
 func NewClusterParams() *ClusterParams {
 	p := &ClusterParams{}
 	p.DefaultFeatureGates = []string{
+		"APIPriorityAndFairness=true",
```

Comment:
we may want to follow-up with @deads2k if we want to actually enable this when we run externalized.

Reply:
checked with @deads2k, the flag should still be helpful in our case. It won't have a negative effect.